affinity/src/main/java/net/openhft/affinity/LockCheck.java

```
@@ -109,15 +109,10 @@ static int getProcessForCpu(int core) throws IOException {
         return EMPTY_PID;
     }
 
-    static void updateCpu(int cpu) {
+    static void updateCpu(int cpu) throws IOException {
         if (!canOSSupportOperation())
             return;
-        try {
-            replacePid(cpu, getPID());
-        } catch (IOException e) {
-            LOGGER.warn("Failed to update lock file for cpu " + cpu, e);
-            e.printStackTrace();
-        }
+        replacePid(cpu, getPID());
```

stop swallowing the exception